client/src/snippets.json

```diff
@@ -183,5 +183,49 @@
             "</xrefs>"
         ],
         "description": "Add a xref section with bio.tools type."
-    }
+    },
+    "single output": {
+        "prefix": "gx-output-single",
+        "body": [
+            "<data name=\"--$1\" format=\"$2\" label=\"${tool.name} on ${on_string}\" />"
+        ],
+        "description": "Add a simple single output."
+    },
+    "single output picked up from the working directory": {
+        "prefix": "gx-output-single-from-work-dir",
+        "body": [
+            "<data name=\"--$1\" format=\"$2\" from_work_dir=\"$3\" label=\"${tool.name} on ${on_string}\" />"
+        ],
+        "description": "Add a simple single output, that is picked up from the job working directory."
+    },
+    "single tabular output with column-metadata": {
+        "prefix": "gx-output-single-tabular-with-metadata",
+        "body": [
+            "<data name=\"--$1\" format=\"tabular\" from_work_dir=\"$2\" label=\"${tool.name} on ${on_string}\">",
+            "    <actions>",
+            "        <action name=\"column_names\" type=\"metadata\" default=\"$3,$4,$5\"/>",
+            "    </actions>"
+        ],
+        "description": "Add a single tabular output and specify the column names as metadata."
+    },
+    "single optional output with filters": {
+        "prefix": "gx-output-single-filter",
+        "body": [
+            "<!-- TODO: filters needs to be adopted, see examples at https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-outputs-data-filter -->",
+            "<data name=\"--$1\" format=\"$2\" from_work_dir=\"$3\" label=\"${tool.name} on ${on_string}\">",
```

Comment:
I'm not sure if `from_work_dir` is something common to have in these... If not better not to include it to avoid having to edit the snippet after insert.

```suggestion
            "<!-- TODO: filters need to be adapted, see examples at https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-outputs-data-filter -->",
            "<data name=\"$1\" format=\"$2\" from_work_dir=\"$3\" label=\"${tool.name} on ${on_string}\">",
```

Reply:
from_work_dir is pretty common I would say....